home/templates/home/user_page.html

```diff
@@ -0,0 +1,26 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+<link rel="stylesheet" type="text/css" href="{% static 'home/CSS/user_page.css' %}">
+<body class="main-bg">
+<main role="main" class="container">
+ <h1 class="title">{{ profile }} Profile page</h1>
+<div class="row">
+  <div class="column left-col border border-dark" >
+   <h2>User Questions</h2>
+   {% for question in user_questions %}
+		<a class="questions" href="{% url 'question-detail' question.pk  %}">{{question}}</a>
+		<br/>
+	{% endfor %}
+  </div>
+  <div class="column right-col border border-dark">
+  <h2>User Answers</h2>
+  {% for answer in user_answers %}
+		<a class="answers" href="{% url 'question-detail' answer.question.pk  %}">Answered {{answer.question}}</a>
+		<br/>
+	{% endfor %}
```

Comment:
Maybe we should put more data on the profile's page besides the user's answers and questions.
I thought about adding as well: num of answers, num of questions, registration date, and email.
What do you think?

Reply:
I'm down to add num of.. and registration date, email is something that should be kept privately IMO, and maybe I can add the gender with an adjacent icon if we already have this info.

Reply:
Ok, it is possible at right now not to display this data and in the future maybe we will add an option to ask the user which data he prefers to be displayed and which not.